Week1/Day3/TimedChallenge1/code.py

```diff
@@ -0,0 +1,7 @@
+reverseinp = input("Enter a sentence: ")
```

Comment:
better name for this variable?

Reply:
user_sentence or just sentence

but "reverseinp" is name that was is the task:

Here is the starter python code :

REverseinp= raw_input()


#write down your logic here